aeson.cabal

```diff
@@ -174,6 +174,9 @@ test-suite tests
   if !impl(ghc >= 7.10)
     build-depends: nats >=1 && <1.2
 
+  if impl(ghc >= 7.8)
+    build-depends: hashable-time >= 0.2 && <0.3
+
```

Comment:
`hashable-time` hss restrictive bounds atm, so some tests are run only when it's available. I could pit Ot behind a automatic flag too, but then it will be harder to spot that tests aren't run at all